### ISSUE TITLE
Parent term change for ATP-dependent DNA/DNA annealing activity

### ISSUE THREAD

**Issue by hattrill** [geneontology/go-ontology#29334]
GO:0036310 ATP-dependent DNA/DNA annealing activity
is_a GO:0140657 ATP-dependent activity

should be moved under : GO:0008094 ATP-dependent activity, acting on DNA

**Comment by pgaudet** [comment]
Added missing parent. 

However I am wonder about this little branch: 
<img width="413" alt="image" src="https://github.com/user-attachments/assets/e98326dd-4ae8-4e7b-9b06-3a2cdc967ee5">


It's under 'binding', but it should be under a 'real' function. Do you think molecular adaptor activity would work? 

<img width="388" alt="image" src="https://github.com/user-attachments/assets/50e3bebd-f317-41ca-a911-b11589189873">


Thanks, Pascale

**Comment by hattrill** [comment]
My electricity is just being shutoff for a couple hrs - so if I am silent for a bit..... But, these annealing activities are more related to helicase-type modality than adaptors, I think.

**Comment by pgaudet** [comment]
Just talking to @colinlog 
How about annealing activity is_a [DNA-DNA tethering activity](http://purl.obolibrary.org/obo/GO_0106260)? 
(which is a child of molecular adaptor too ;)

**Comment by hattrill** [comment]
I don't think that is correct as it is a very transient thing - nothing is being bridged but more smooshed together. Can is sit under 'catalytic activity, acting on a nucleic acid'?

**Comment by ValWood** [comment]
I never understood this term. There are 10 EXP annotations.
Naive question: if you unwind a helicase  and form a bubble, the DNA must reanneals automatically, right? why is a catalytic activity required? The base bairs are right in proximity? So what does it really mean?

**Comment by ValWood** [comment]
CHAT-GPT
DNA reannealing activity refers to the process by which single-stranded DNA molecules hybridize or bind together to form double-stranded DNA. This occurs when complementary base sequences in the single strands find each other under suitable conditions, such as the right temperature, ionic strength, and time.

Q is it a catalytic activity?
No, DNA reannealing activity is not considered a catalytic activity. Here's why:

Catalytic Activity Defined:

Catalytic activities involve enzymes or catalysts that speed up a chemical reaction without being consumed or altered in the process.
These activities usually involve enzymes with active sites specifically designed to lower the activation energy of a reaction.
DNA Reannealing:

DNA reannealing is a physical process rather than a chemical one. It occurs when complementary single-stranded DNA molecules find each other and hydrogen bond to form a double-stranded structure.
The process depends on physical factors like temperature, DNA concentration, and ionic conditions rather than enzymatic facilitation.
Enzymatic Facilitation in Related Processes:

While DNA reannealing itself is not catalytic, some enzymes, like helicases, may influence DNA strand separation and reannealing indirectly by affecting the single-stranded state.
In processes like homologous recombination, proteins such as RecA (in bacteria) or RAD51 (in eukaryotes) can facilitate strand pairing. These are considered catalytic in the sense of promoting DNA strand exchange but are not directly responsible for reannealing itself.
In summary, DNA reannealing is a biophysical phenomenon, not a catalytic activity, though it can be influenced by certain enzymes in specific biological contexts.

**Comment by hattrill** [comment]
As some require ATP hydrolysis, does not look passive. Could treat it as a class of chaperone - although this is somewhat vague. I am amenable to some sort of 'chaperone' class.

[annealing helicase](https://pmc.ncbi.nlm.nih.gov/articles/PMC3409536/)

AI Overview
Annealing helicases are a class of enzymes that can anneal two complementary single-stranded nucleic acids, while lacking detectable unwinding activity. They use ATP to rewind DNA in an ATP-dependent manner.

**Comment by ValWood** [comment]
In these cases, is the ATP hydrolysis definitely connected to the reannealing because the ones I looked at seemed to have unwinding activity, too?

**Comment by hattrill** [comment]
No problem - I have had exactly the same concerns.

**Comment by ValWood** [comment]
When I look at the 29 EXP annotations for DNA/DNA annealing activity annealing,
most do seem to also be helicases i.e
RECQL4
BLM
RECQL3


or have have some other activity
SMARCA1 is an ATP dependent chromatin remodeller

RAD52 might be an exception but is the annealing a consequence of some other activity?

POmbase  has annotations for cut14 , cut3, and cnd3 with contributes to

**Comment by ValWood** [comment]
POmbase has annotations for cut14 , cut3, and cnd3 with contributes to i"ll look at these

In this study, we employed an in vitro approach, building upon our previous study, which demonstrated the ability of the condensin SMC heterodimer Cut3–Cut14 to remove single-stranded (ss) DNA-binding protein RPA or Ssb1, which had been bound to the unwound ssDNAs [[12](https://royalsocietypublishing.org/doi/10.1098/rsob.140193#RSOB140193C12)]. As the elimination of protein and/or RNA during the re-annealing reaction per se did not require ATP, we wondered how ATP interacts with condensin's ATPase domain during the re-annealing reaction. To our great surprise, we discovered that the phosphate groups of ATP bind to multiple sites on Cut3, an SMC4-like subunit, but this apparent ‘auto-phosphorylation’ is greatly diminished when the holocondensin complex, which has ample ATPase activity, is employed. We show that multiple phosphorylation sites are located in the hinge of Cut3/SMC4, and that phosphorylation is abolished in ATPase mutants. We investigated this ATPase-dependent phosphorylation of the hinge in detail; we propose that hinge phosphorylation represents a step in condensin's ATPase cycle, and that it is important for understanding condensin's dissociation from chromosomal DNA. In other words, ATP enables the mobility of condensin along chromosomes by causing it to dissociate from DNA.

3.16. ATPγS-pretreated wild-type SMC, but not ATPase mutant SMC, fails DNA re-annealing
To obtain information about possible functional changes in Cut3–Cut14 after phosphorylation, we tested whether wild-type and ATPase mutant Cut3–Cut14, pretreated with ATPγS, were able to re-anneal hdDNA into dsDNA (Material and methods, [figure 5](https://royalsocietypublishing.org/doi/10.1098/rsob.140193#RSOB140193F5)a). Wild-type Cut3–Cut14 and ATPase mutant Cut3K161I–Cut14K38T complex were pre-incubated with ATPγS or ADP for 2 h at 30°C, and then passed through α Dye Ex column (QIAGEN) that eliminated nucleotides. Resulting heterodimer samples were incubated with hdDNA to examine their DNA-re-annealing ability ([figure 5](https://royalsocietypublishing.org/doi/10.1098/rsob.140193#RSOB140193F5)b). Wild-type Cut3–Cut14 treated with or without ADP could re-anneal hdDNA, whereas wild-type Cut3–Cut14 treated with ATPγS failed to do so, showing that thiophosphorylation rendered Cut3–Cut14 dysfunctional for DNA re-annealing. ADP treatment had no inhibitory effect regarding DNA re-annealing activity, however.
By contrast, the ATPase double mutant Cut3 K161I–Cut14 K38T protein treated with ATPγS could promote re-annealing ([figure 5](https://royalsocietypublishing.org/doi/10.1098/rsob.140193#RSOB140193F5)c), consistent with the notion that the intact ATPase domain is not required for DNA re-annealing. Actually, the intact domain appeared to be needed for the ATPγS-induced loss of re-annealing ability.

Here it seems like the mutants affect reannealing ability. Not that reannealing is an'activity'

**Comment by hattrill** [comment]
Interesting. 
I've only come across helicases that have this activity. We currently only have two genes with this annotation:
Marcal1 - which has the summary 'encodes an ATP-dependent DNA annealing helicase that recognizes single-stranded to double-stranded DNA transitions. It maintains genomic integrity through its roles in DNA repair, DNA replication, DNA recombination and gene expression.'
 Blm (Bloom syndrome helicase) - which has annealing and unwinding helicase annotations - appears to do these activities "combines DNA strand displacement with DNA strand annealing to catalyze the displacement of one DNA strand while annealing a second complementary DNA strand" and the ATP binding might influence strand binding activities.

**Comment by sjm41** [comment]
I've just made some new terms in the isomerase branch to align GO with EC - see https://github.com/geneontology/go-ontology/issues/29690

So helicase activity now has this parentage:

```
GO:0016853 isomerase activity (EC:5.-.-.-)
    |_GO:0120543 macromolecular conformation isomerase activity (EC:5.6.-.-)
        |_GO:0120545 nucleic acid conformation isomerase activity (EC:5.6.2.-)
            |_GO:0004386 helicase activity (children include specific helicase terms corresponding to 5.6.2.3, 5.6.2.4, 5.6.2.5, 5.6.2.6)
```

The 'annealing activity' tree currently looks like this:
```
molecular_function
   |__binding
       |__organic cyclic compound binding
           |__nucleic acid binding
               |__annealing activity
                   |__DNA/DNA annealing activity
                       |__ATP-dependent DNA/DNA annealing activity
                   |__DNA/RNA hybrid annealing activity
                   |__RNA strand annealing activity
```

I think 'ATP-dependent DNA/DNA annealing activity' (which is used to annotate fly Marcal1/human SMARCAL1, amongst others) should have an additional parent of GO:0120545 nucleic acid conformation isomerase activity'
Sound OK?

id: GO:0036310
name: ATP-dependent DNA/DNA annealing activity
def: "An ATP-dependent activity that facilitates the formation of a complementary double-stranded DNA molecule." [PMID:21078962, PMID:22704558, PMID:22705370, PMID:22759634, PMID:22888405]
is_a: GO:0008094 ! ATP-dependent activity, acting on DNA
is_a: GO:1990814 ! DNA/DNA annealing activity
relationship: has_part GO:0016887 ! ATP hydrolysis activity

id: GO:0120545
name: nucleic acid conformation isomerase activity
def: "Catalysis of a reaction that alters the conformation of a nucleic acid." [EC:5.6.2.-]


Alternatively, we could add the GO:0120545 nucleic acid conformation isomerase activity higher in the tree, maybe as a parent to 'annealing activity'?? But I'm not sure that's appropriate from the discussion above.

id: GO:0140666
name: annealing activity
def: "An activity that facilitates the formation of a complementary double-stranded polynucleotide molecule." [PMID:22888405]

**Comment by hattrill** [comment]
Thanks @sjm41, think the latter option where it is higher in the tree is the safest.
GO:0140666 annealing activity is_a GO:0120545 nucleic acid conformation isomerase activity

**Comment by sjm41** [comment]
OK, I'll do that.

**Comment by sjm41** [comment]
As suggested above, I did this:
> GO:0140666 annealing activity is_a GO:0120545 nucleic acid conformation isomerase activity

Which resulted in the following tree (on which I'm showing the 6 Dmel annotations in FB):

```
catalytic activity
    |__macromolecular conformation isomerase activity
         |__nucleic acid conformation isomerase activity
             |__annealing activity
                 |__DNA/DNA annealing activity (RecQ4, Marcal1, Blm)
                     |__ATP-dependent DNA/DNA annealing activity (RecQ4, Marcal1)
                 |__DNA/RNA hybrid annealing activity
                 |__RNA strand annealing activity (eIF4H1, eIF4H2, CG3594)
```

So we now have all types of 'annealing activity' in the catalytic activity branch.

This includes 'RNA strand annealing activity' that we have annotated to fly eIF4H1, eIF4H2 and CG3594 (distant relation to eIF4H) via [PANTHER:PTN008594436](https://www.pantree.org/node/annotationNode.jsp?id=PTN008594436) (which is based on annotation to yeast TIF3 = P34167). 


Reviewing the comments above, I see that Val's AI responses said that '**DNA reannealing activity is not a catalytic activity**' whereas Helen's AI response (which was specifically about ATP-dependent annealing helicases) said '**annealing helicases are a class of enzymes that can anneal two complementary single-stranded nucleic acids, while lacking detectable unwinding activity. They use ATP to rewind DNA in an ATP-dependent manner.**'

So I'm now wondering whether it would be more accurate just to have the 'ATP-dependent DNA/DNA annealing activity' term under 'catalytic activity'.
I don't think eIF4H proteins are considered enzymes with RNA annealing activity, but maybe this is an issue with the PAINT annotation rather than with the ontology.

Any further thoughts from you @ValWood @hattrill @pgaudet ?

**Comment by hattrill** [comment]
Not particularly wedded to either option. IMO - ok as it is, they are all bonding reactions that are 'facilitated'.

**Comment by sjm41** [comment]
OK, I'll leave the ontology alone.

But I do think this is wrong:

>This includes 'RNA strand annealing activity' that we have annotated to fly eIF4H1, eIF4H2 and CG3594 (distant relation to eIF4H) via [PANTHER:PTN008594436](https://www.pantree.org/node/annotationNode.jsp?id=PTN008594436) (which is based on annotation to yeast TIF3 = P34167).

because:
- Yeast TIF3  appears to most similar to fly/human eIF4B, not eIF4H (Phylome is the only orthology algorithm that says eIF4H and TIF3 are orthologs.)
- Indeed, the source publication for the yeast TIF3 annotation has the title "The Saccharomyces cerevisiae translation initiation factor Tif3 and its mammalian homologue, eIF-4B, have RNA annealing activity."

So I'll make a ticket for Marc in the other tracker about this.

**Comment by sjm41** [comment]
https://github.com/geneontology/go-annotation/issues/5611#issuecomment-2824116440
